parm/snow/obs/lists/gdas_snow.yaml.j2

```diff
@@ -0,0 +1,9 @@
+observers:
+{% filter indent(width=2) %}
+{% include 'snow/obs/config/adpsfc_snow.yaml.j2' %}
+{% include 'snow/obs/config/snocvr_snow.yaml.j2' %}
+{% set cycle = current_cycle | strftime("t%Hz") %}
+{% if cycle in ['t18z'] %}
+{% include 'snow/obs/config/ims_snow.yaml.j2' %}
+{% endif %}
+{% endfilter %}
```

Comment:
No EOL here, not sure how big of a deal that is?

Reply:
shouldn't be an issue, but added EOL.